examples/socket.io-chat-app/index.js

```diff
@@ -0,0 +1,20 @@
+const micro = require('micro'),
+      fs = require('fs');
+
+const server = micro(async (req, res) => {
+  const html = fs.readFileSync(__dirname + '/index.html');
```

Comment:
Hum yes did not think of that, I used it naively to serve the html but you're right it will just create troubles. What would be the right way to do it with micro ?